pkg/webservice/handlers_test.go

```diff
@@ -443,3 +444,105 @@ func prepareSchedulerForConfigChange(t *testing.T) {
 		t.Fatalf("Error when load clusterInfo from config %v", err)
 	}
 }
+
+func TestGetNodesUtilJSON(t *testing.T) {
+	const configDefault = `
+partitions:
+  - name: default
+    queues:
+      - name: root
+        queues:
+        - name: default
+`
+	rmID := "Util"
+	policyGroup := "default-policy-group"
+	clusterInfo := cache.NewClusterInfo()
+
+	configs.MockSchedulerConfigByData([]byte(configDefault))
+	if _, err := cache.SetClusterInfoFromConfigFile(clusterInfo, rmID, policyGroup); err != nil {
+		t.Fatalf("Error when load clusterInfo from config %v", err)
+	}
+	assert.Equal(t, 1, len(clusterInfo.ListPartitions()))
+
+	// Check test partition
+	partitionName := "[" + rmID + "]default"
+	partition := clusterInfo.GetPartition(partitionName)
+	assert.Equal(t, partitionName, partition.Name)
+	// create test application
+	queueName := "root.default"
+	appInfo := cache.CreateNewApplicationInfo("appID-1", "default", queueName)
+	err := cache.AddNewApplicationForTest(partition, appInfo, true)
+	if err != nil {
+		t.Errorf("add application to partition should not have failed: %v", err)
+	}
+
+	memVal := resources.Quantity(1000)
+	coreVal := resources.Quantity(1000)
+	// create test nodes
+	node1 := cache.NewNodeForTest("node-1", resources.NewResourceFromMap(
+		map[string]resources.Quantity{"memory": memVal, "vcore": coreVal}))
```

Comment:
please use MEMORY and VCORE constants defined in resources.go

Reply:
done